packages/web/src/components/Admin/ExpoJudgingSessionForm/ExpoJudgingSessionForm.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-underscore-dangle */
+import { Button, Flex, FormControl, FormLabel } from '@chakra-ui/react';
+import React from 'react';
+import { useExpoJudgingSessionForm } from './useExpoJudgingSessionForm';
+import { ProjectsSelect } from '../../ProjectsSelect';
+import { FormHelperHint } from '../../Forms';
+
+type ExpoJudgingSessionFormProps = {};
+
+export const ExpoJudgingSessionForm: React.FC<ExpoJudgingSessionFormProps> = () => {
+  const { isLoading, formik, errors } = useExpoJudgingSessionForm();
+  // const inputStyleProps: Pick<InputProps | TextareaProps, 'variant' | 'isDisabled'> = {
+  //   variant: 'filled',
+  //   isDisabled: isLoading,
+  // };
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <Flex w="full" direction="column" gap={5}>
+        {/* <FormControl id="title">
```

Comment:
are there any plans to revive this dead code in the near future?

Reply:
Yep, as soon as Ansaar finishes one of his current stories we can comment all of this back in and it should work out of the box